src/shared/components/nav-icons/nav-icons.js

```diff
@@ -15,31 +16,31 @@ const NavIcons = () => {
             <a href="/authentication/signin/signin.html" className="text-xs">
               Login
```

Comment:
Can use Link here

Reply:
Yes I will change when I will apply functionality of login